docs/core_docs/docs/expression_language/cookbook/with_history.mdx

```diff
@@ -0,0 +1,6 @@
+import CodeBlock from "@theme/CodeBlock";
+import Example from "@examples/guides/expression_language/runnable_history.ts";
+
+# Runnables With History
+
+<CodeBlock language="typescript">{Example}</CodeBlock>
```

Comment:
Would call this `Runnables With Chat History` and remove the `Memory` section from cookbook sidebar (don't remove page)